rllib/policy/eager_tf_policy.py

```diff
@@ -729,12 +733,35 @@ def set_state(self, state: PolicyState) -> None:
             super().set_state(state)
 
         @override(Policy)
-        def export_checkpoint(self, export_dir):
-            raise NotImplementedError  # TODO: implement this
-
-        @override(Policy)
-        def export_model(self, export_dir):
-            raise NotImplementedError  # TODO: implement this
+        def export_model(self, export_dir, onnx: Optional[int] = None) -> None:
+            if onnx:
+                try:
+                    import tf2onnx
+                except ImportError as e:
+                    raise RuntimeError(
+                        "Converting a TensorFlow model to ONNX requires "
+                        "`tf2onnx` to be installed. Install with "
+                        "`pip install tf2onnx`."
+                    ) from e
+
+                model_proto, external_tensor_storage = tf2onnx.convert.from_keras(
+                    self.model.base_model,
```

Comment:
do we enforce the name of ``base_model`` somehow?
how does this work for custom models? e.g., the model that Riot uses?

Reply:
For tf, the sad truth is that it won't work right now. TfModelV2 is NOT a keras model (which is stupid, it should be).
The idea is to get us into the RLModules/RLTrainers design as fast as possible and then this problem will go away, rather than finding some shim solution for right now.

Reply:
can you add a detailed comment about this? maybe in the doc string, laying out the limitation of current implementation?
if this doesn't work for tf2, what's the plan for Riot ... ?